homeassistant/components/sensor/yr.py

```diff
@@ -185,7 +187,8 @@ def try_again(err: str):
                 model = self.data['meta']['model']
                 if '@nextrun' not in model:
                     model = model[0]
-                self._nextrun = dt_util.parse_datetime(model['@nextrun'])
+                self._nextrun = dt_util.parse_datetime(model['@nextrun']) +\
```

Comment:
If we want to extend this time we should abort the update cycle and reschedule in 15 minutes rather. (Almost like a web request failing)